README.md

```diff
@@ -18,6 +18,65 @@ And then execute:
 
 ### Usage
 
+Before starting this, you'll need to
+
+- get your cookie listed on [/help/cookies](https://www.gov.uk/help/cookies)
```

Comment:
@tomnatt suggested that we update the help page to cover all `ABTest-*` cookies so that we can skip this step in future.